driver-core/src/main/com/mongodb/internal/operation/ClientBulkWriteOperation.java

```diff
@@ -180,23 +194,79 @@ public ClientBulkWriteResult execute(final WriteBinding binding) throws ClientBu
         return resultAccumulator.build(transformedTopLevelError, effectiveWriteConcern);
     }
 
+    @Override
+    public void executeAsync(final AsyncWriteBinding binding,
+                             final SingleResultCallback<ClientBulkWriteResult> finalCallback) {
+        WriteConcern effectiveWriteConcern = validateAndGetEffectiveWriteConcern(binding.getOperationContext().getSessionContext());
+        ResultAccumulator resultAccumulator = new ResultAccumulator();
+
```

Comment:
The sync counterpart does not have this empty line.

```suggestion
```

Reply:
As discussed on the call, instead of removing the line breaks from the async implementation, I’ve added the same line breaks to the sync version for consistency.